### ISSUE TITLE
Make a set of JWT decoding keys public

### ISSUE THREAD

**Issue by slavik-pastushenko** [chenhunghan/jwks#5]
Hello!

Seems like the provided example does not work.

I am not able to access the `keys`.
The issues are: 
- `field "keys" of Jwks is private`.
- `field "decoding_key" of Jwk is private`.

is it possible to make the `keys` property public?

```
/// A container for a set of JWT decoding keys.
///
/// The container can be used to validate any JWT that identifies a known key
/// through the `kid` attribute in the token's header.
#[derive(Clone)]
#[allow(dead_code)]
pub struct Jwks {
    pub keys: HashMap<String, Jwk>,
}
```

The same for 
```
pub struct Jwk {
    pub decoding_key: DecodingKey,
}
```

**Comment by slavik-pastushenko** [comment]
I was trying to make a PR but unfortunately I am not able to push a new branch :(

**Comment by chenhunghan** [comment]
Thanks for reporting, let's make it work :)

**Comment by chenhunghan** [comment]
@slavik-pastushenko should be fixed now in 0.1.3, feel free to fork and send a pull request if you find anything :)

**Comment by slavik-pastushenko** [comment]
thanks a lot!
